book/src/building/xcode-and-cargo/README.md

```diff
@@ -85,18 +85,24 @@ if [[ -n "${DEVELOPER_SDK_DIR:-}" ]]; then
   export LIBRARY_PATH="${DEVELOPER_SDK_DIR}/MacOSX.sdk/usr/lib:${LIBRARY_PATH:-}"
 fi
 
+if [[ $LLVM_TARGET_TRIPLE_SUFFIX == "-simulator" ]]; then
```

Comment:
Just so that we don't pollute the environment vars of sub-processes:

```sh
TARGETS=""
if [ "$PLATFORM_NAME" = "iphonesimulator" ]; then
  TARGETS="..."
else
  TARGETS="..."
fi
```

Reply:
Yes, you are right!